sort-glossary.py

```diff
@@ -0,0 +1,32 @@
+"""
+Script to sort the glossary entries.
+
+Usage: python sort-glossary.py
+"""
+
+GLOSSARY_PATH = 'docs/89_glossary.md'
+
+def main():
+    with open(GLOSSARY_PATH, encoding='utf-8') as f:
+        text = f.read()
+
+    sep = '\n\n* **'
+    parts = text.split(sep)
+    # Normalize the amount of space between entries.
+    parts = [part.strip() for part in parts]
+
+    # Keep the intro at the beginning.
+    first_part = parts[0]
+    parts = sorted(parts[1:])
```

Comment:
I think that this should probably read `sorted(parts[1:], key=lambda s: s.lower())` and declare the file as Python2.